plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/LibraryFactory.java

```diff
@@ -134,8 +137,14 @@ private static MavenCoordinates getMavenCoordinates(IConfigurationElement[] chil
     if (!Strings.isNullOrEmpty(repository)) {
       mavenCoordinates.setRepository(repository);
     }
-    
+
+    // Only look up latest version if version isn't specified in file.
     String version = mavenCoordinatesElement.getAttribute(ATTRIBUTE_NAME_VERSION);
+    if (Strings.isNullOrEmpty(version) || "LATEST".equals(version)) {
+      // todo don't load beta versions
+      version = retriever.getLatestArtifactVersion(groupId, artifactId).toString(); 
```

Comment:
The (original) Javadoc of `getLatestArtifactVersion` says it can potentially return `null`:

> Returns the latest published artifact version, or null if there is no such version.

However it is not clear to me how there can be no published version at all, other than providing wrong coordinates (in which case, the `pom.xml` is just wrong?)